source/agora/consensus/EnrollmentPool.d

```diff
@@ -320,28 +327,31 @@ unittest
         tx.inputs[0].signature = signature;
         storage.put(tx);
     }
-    EnrollmentPool pool = new EnrollmentPool(":memory:");
+    auto pool = new EnrollmentPool(":memory:", new immutable(ConsensusParams)());
     Hash[] utxo_hashes = storage.keys;
 
     // add enrollments
     Scalar[Hash] seed_sources;
     auto utxo_hash = utxo_hashes[0];
     seed_sources[utxo_hash] = Scalar.random();
-    auto enroll = createEnrollment(utxo_hash, key_pair, seed_sources[utxo_hash]);
+    auto enroll = createEnrollment(utxo_hash, key_pair, seed_sources[utxo_hash],
+        pool.params.ValidatorCycle);
```

Comment:
Yeah this is unrelated to the pool. The longevity of a validator's cycle is unrelated to how long an enrollment stays in the pool.

Reply:
I can change it with a `ConsensusParams' object. Thanks.